app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,5 @@
+class CategoriesController < ApplicationController
```

Comment:
Is this controller used for anything?

Reply:
Valid point, didnt need it in the end. Removed